src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -815,9 +813,7 @@ public ResponseEntity<String> getVoltageInitResult(@Parameter(description = "stu
 
     @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/voltage-init/status")
     @Operation(summary = "Get the voltage init status on study")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The voltage init status"),
-            @ApiResponse(responseCode = "204", description = "No voltage init has been done yet"),
-            @ApiResponse(responseCode = "404", description = "The voltage init status has not been found")})
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The voltage init status"), @ApiResponse(responseCode = "204", description = "No voltage init has been done yet"), @ApiResponse(responseCode = "404", description = "The voltage init status has not been found")})
```

Comment:
no need to this change

Reply:
It's required by CheckStyle otherwise the build fail.